cypress/e2e/inventory/holdings/check-new-formatting-statistical-code-on-holding-screen.cy.js

```diff
@@ -0,0 +1,125 @@
+import { including } from 'bigtest';
```

Comment:
please use import { including } from '@interactors/html'; instead of import { including } from 'bigtest';

Reply:
done